src/resources/parser/glb-parser.js

```diff
@@ -1782,6 +1783,24 @@ const createLight = function (gltfLight, node) {
         lightProps.outerConeAngle = gltfLight.spot.hasOwnProperty('outerConeAngle') ? gltfLight.spot.outerConeAngle * math.RAD_TO_DEG : Math.PI / 4;
     }
 
+    // glTF stores light already in energy/area, but we need to provide the light with only the energy parameter,
```

Comment:
We already have the same code to compute coefficients in the Light class. There we divide by them, here we multiply. We'll also need them in the gltf-exporter soon. And so I wonder if those should be in a place where we can share them.
Just not sure where to put them. Perhaps a static function on the Light class?

Reply:
That's a good point actually. Since they basically just act as multipliers, we could move them to somewhere shared. I'll do that 🙂